packages/govern-console/src/containers/NewProposal/NewProposal.tsx

```diff
@@ -279,7 +278,7 @@ const NewProposal: React.FC<NewProposalProps> = ({ onClickBack, ...props }) => {
   const { account, provider, isConnected } = context;
 
   const proposalInstance = React.useMemo(() => {
-    if (provider && account && daoDetails) {
+    if (provider && daoDetails && account.address && account.signer) {
```

Comment:
any specific reason why writing account.address && account.signer was needed instead of account directly ?